src/test/java/HomepageTest.java

```diff
@@ -1,45 +1,146 @@
-import org.assertj.core.api.Assertions;
-import org.openqa.selenium.By;
+import org.assertj.core.api.SoftAssertions;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 import pages.Homepage;
 import utils.WebDriverUtils;
 
-import java.time.Duration;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
 public class HomepageTest {
 
     private WebDriver driver;
 
+    private static final String WHITE_COLOR = "#ffffff";
+    private static final String BLUE_COLOR = "#1979c3";
+
     @BeforeClass
     public void setUp() {
         driver = WebDriverUtils.createWebDriver();
     }
 
+    @BeforeMethod
+    public void beforeMethod() {
+        driver.get("https://magento.softwaretestingboard.com/");
+    }
+
     @Test
     public void homepage_MainBanner_HasTextAndButtonCorrectlyDisplayed() {
         //given
         Homepage homepage = new Homepage(driver);
```

Comment:
This part - creating homepage - is done in each test method.
But it's not needed. Should be resolved in the same way lik in LoginFormTest.
The class field should be defined private Homepage homepage and then created after page opening in the setUp() method.

There needs to be remembered that creating objects in every test method takes memory. The more memory is taken and more object is created, the code starts to be bigger and bigger and affects tests performance.